script.js

```diff
@@ -1,4 +1,280 @@
-var main = function (input) {
-  var myOutputValue = 'hello world';
+let main = function (input) {
+  console.log('--------- input field', input)
+  let myOutputValue =playRPS2(input);
+  // let myOutputValue =playRPS(input);
+  // let myOutputValue =normalRPS(input);
+  // let myOutputValue =reverseRPS(input);
   return myOutputValue;
 };
+
+let Rock = 'rock';
+let Paper = 'paper';
+let Scissor = 'scissors';
+
+let userName = ''
+let gameType = ''
+
+// includes reversed for Part 1
+// let Reversed_Rock = 'reversed rock';
+// let Reversed_Paper = 'reversed paper';
+// let Reversed_Scissor = 'reversed scissors';
+
+let replayMessage = `Now you can type "scissors" "paper" or "stone" to play another round!  <br><br> Total Games played is `
+let currentMode = 'waiting for user name';
+// let gameMode = 'normal game mode';
+let countGamesPlayed = 0;
+let countComputerWins = 0;
+let countGamesDraw = 0;
+
+// 1. Generate RPS sign
+let getComputerSign = function ()  {
+    let randomNumber = (Math.floor(Math.random() * 3))
+    if (randomNumber === 0){
+      return Rock;
+    }
+    if (randomNumber === 1){
+      return Paper;
+    }
+    if (randomNumber === 2){
+    return  Scissor;
+    }
+    console.log(getComputerSign());
```

Comment:
this would console log another run of getComputerSign() which won't be the output from the current run of getComputerSign(). 
this will not run anyways cos the function will be returned before the code runs line 41. 
If you wish to console log, you can initialise 
var computerSign=' ' // after line 31
and reassign computerSign  in each if-code-block.
good job at making this a function tho!